sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.cs

```diff
@@ -16,6 +16,13 @@ namespace Azure.Storage.Test.Shared
 {
     public abstract class StorageTestBase : RecordedTestBase
     {
+        static StorageTestBase()
+        {
+            // https://github.com/Azure/azure-sdk-for-net/issues/9087
+            // .NET framework defaults to 2, which causes issues for the parallel upload/download tests.
+            ServicePointManager.DefaultConnectionLimit = 100;
```

Comment:
Should we condition this on .NET Framework?  I think this is effectively a no-op in .NET Core, but I would also prefer to keep .NET Core as default as possible.